src/test/resources/features/Scoring/Report/Report.feature

```diff
@@ -0,0 +1,130 @@
+Feature: Report Feature
+
+  Background:
+    Given user prepare auth request
+    And user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    And user prepare report request with batchCode "futur3"
```

Comment:
I found it best to always call `And user hit logout endpoint` in each `Background`, since previous tests may alter the state of `authData` singleton